File: Demos/Visual Basic .Net/09. Addendas/Adenda/It Smart Business.vb
Imports HyperSoft.Ejemplo.Data
Imports HyperSoft.ElectronicDocumentLibrary.Itsb.Addenda

Friend NotInheritable Partial Class Adenda
	
	Friend Shared Function ItSmartBusiness(ByRef fileName As String) As Boolean
		'En este método se cargan los datos de la factura.
		Cfdi40.CargarDatosCompleto(electronicDocument)

		Dim addenda As ItSmartBusiness = New HyperSoft.ElectronicDocumentLibrary.Itsb.Addenda.ItSmartBusiness()

		addenda.Data.Itsb.Version.Value = "1.1"
		addenda.Data.Itsb.OrdenCompra.Value = "PO-0000000117"


		electronicDocument.Data.Addendas.Add(addenda)

		Return Save("Addenda_ItSmartBusiness.xml", fileName)
	End Function

End Class

File: Demos/CSharp/09. Addendas/Adenda/It Smart Business.cs
using HyperSoft.Ejemplo.Data;
using HyperSoft.ElectronicDocumentLibrary.Itsb.Addenda;

namespace HyperSoft.Ejemplo.Adendas
{
  internal static partial class Adenda
  {
    internal static bool ItSmartBusiness(out string fileName)
    {
      //En este método se cargan los datos de la factura.
      Cfdi40.CargarDatosCompleto(electronicDocument);

      ItSmartBusiness addenda = new HyperSoft.ElectronicDocumentLibrary.Itsb.Addenda.ItSmartBusiness();

      addenda.Data.Itsb.Version.Value = "1.1";
      addenda.Data.Itsb.OrdenCompra.Value = "PO-0000000117";


      electronicDocument.Data.Addendas.Add(addenda);

      return Save("Addenda_ItSmartBusiness.xml", out fileName);
    }
  }
}